cli/cmd/simulation/quickexact.hpp

```diff
@@ -10,6 +10,7 @@
 #include <fiction/algorithms/simulation/sidb/sidb_simulation_result.hpp>
 #include <fiction/traits.hpp>
 #include <fiction/types.hpp>
+#include <fiction/utils/layout_utils.hpp>
 #include <fiction/utils/name_utils.hpp>
 
 #include <alice/alice.hpp>
```

Comment:
Not sure why it says this. I think `alice.hpp` can be found fine, as we'll see when all the checkmarks go green once I added back the missing files.

I did notice now there's an import I added at some point that should go since it's not in use. I fixed that in an upcoming push